### ISSUE TITLE
MC crashes when an adapter is placed near a radar station

### ISSUE THREAD

**Issue by logan2611** [ArjanSeijs/OC_ICBM_Addon#4]
ICBM-Classic Version: 4.0.1b75
OC Version: 1.7.5.192
```
java.lang.NoClassDefFoundError: icbm/classic/content/machines/radarstation/TileRadarStation
    at com.eternalsoap.icbmopencomputersaddon.drivers.RadarDriver.getTileEntityClass(RadarDriver.java:17)
    at li.cil.oc.api.prefab.DriverSidedTileEntity.worksWith(DriverSidedTileEntity.java:18)
    at li.cil.oc.server.driver.Registry$$anonfun$2.apply(Registry.scala:102)
    at li.cil.oc.server.driver.Registry$$anonfun$2.apply(Registry.scala:102)
    at scala.collection.TraversableLike$$anonfun$filterImpl$1.apply(TraversableLike.scala:259)
    at scala.collection.mutable.ResizableArray$class.foreach(ResizableArray.scala:59)
    at scala.collection.mutable.ArrayBuffer.foreach(ArrayBuffer.scala:48)
    at scala.collection.TraversableLike$class.filterImpl(TraversableLike.scala:258)
    at scala.collection.TraversableLike$class.filter(TraversableLike.scala:270)
    at scala.collection.AbstractTraversable.filter(Traversable.scala:104)
    at li.cil.oc.server.driver.Registry$.driverFor(Registry.scala:102)
    at li.cil.oc.api.Driver.driverFor(Driver.java:129)
    at li.cil.oc.common.tileentity.Adapter.neighborChanged(Adapter.scala:97)
    at li.cil.oc.common.tileentity.Adapter$$anonfun$neighborCha4096nged$2.apply(Adapter.scala:156)
    at li.cil.oc.common.tileentity.Adapter$$anonfun$neighborChanged$2.apply(Adapter.scala:155)
    at scala.collection.IndexedSeqOptimized$class.foreach(IndexedSeqOptimized.scala:33)
    at scala.collection.mutable.ArrayOps$ofRef.foreach(ArrayOps.scala:186)
    at li.cil.oc.common.tileentity.Adapter.neighborChanged(Adapter.scala:155)
    at li.cil.oc.common.block.Adapter.neighborChanged(Adapter.scala:25)
    at net.minecraft.block.state.BlockStateContainer$StateImplementation.neighborChanged(BlockStateContainer.java:481)
    at net.minecraft.world.World.neighborChanged(World.java:551)
    at net.minecraft.world.World.notifyNeighborsOfStateChange(World.java:497)
    at net.minecraft.world.World.notifyNeighborsRespectDebug(World.java:440)
    at net.minecraft.world.World.markAndNotifyBlock(World.java:381)
    at net.minecraft.world.World.setBlockState(World.java:361)
    at net.minecraft.world.World.setBlockState(World.java:425)
    at icbm.classic.content.blocks.radarstation.TileRadarStation.update(TileRadarStation.java:117)
    at net.minecraft.world.World.redirect$zbe000$tileEntityUpdate(World.java:4137)
    at net.minecraft.world.World.updateEntities(World.java:1838)
    at net.minecraft.world.WorldServer.updateEntities(WorldServer.java:613)
    at net.minecraft.server.MinecraftServer.updateTimeLightAndEntities(MinecraftServer.java:767)
    at net.minecraft.server.MinecraftServer.tick(MinecraftServer.java:668)
    at net.minecraft.server.integrated.IntegratedServer.tick(IntegratedServer.java:185)
    at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526)
    at java.lang.Thread.run(Thread.java:748)
```

**Comment by logan2611** [comment]
Same thing happens with the other blocks as well as it turns out, seems that ICBM-Classic changed the paths of where these files are

**Comment by ArjanSeijs** [comment]
Will have a fix tomorrow,
thanks for reporting!

**Comment by ArjanSeijs** [comment]
Resolved in #5 new version is uploaded to curse and awaiting approval
